doc/modules/ROOT/pages/hacking_on_nrepl.adoc

```diff
@@ -40,3 +40,14 @@ specific version of Clojure like this:
 ----
 $ lein with-profile 1.9 test
 ----
+
+== Logging nREPL messages
+
+You can start the server with the verbose flag set to true, there's
+a default logger that logs the input and output messages with `log/info`,
+but a op `logger` is available so you can set the field `logger` to some
+namespaced symbol representing a function.
+
+See more at test `logger` at `nrepl.core-test`.
```

Comment:
You might also include some sample output from the log, so users would know what to expect.